### PR TITLE
Noto Sans Mono: Version 2.011 added

### PR DESCRIPTION
**Pull request by simoncozens**
04c5f4d: [gftools-packager] Noto Sans Mono: Version 2.011 added

* Noto Sans Mono Version 2.011 taken from the upstream repo https://github.com/notofonts/latin-greek-cyrillic.git at commit https://github.com/notofonts/latin-greek-cyrillic/commit/.

### CODE REVIEW

**Comment by simoncozens** [comment]
Expected changes:

```
Noto Sans Mono v2.011

This release:
* Fixes soft-dotting of i and j glyphs with accents (#64, thanks to Denis Jacquerye)
* Removes the erroneous U+11AB0/11AB1 glyphs (#405)
* Makes integral top and bottom mirrored (#200)
* Aligns integral sign pieces (#201)
* Adds the U+10FB GEORGIAN PARAGRAPH SEPARATOR character for medievalists (#37)
* Adds some sidebearings to MODIFIER LETTER UNASPIRATED (#202)
* Adds the U+2183 ROMAN NUMERAL REVERSED ONE HUNDRED character (#129)
```

**Review by m4rc1e — DISMISSED** [review]
Some math glyphs are broken. Seems to be caused by either the forward slash or fraction.

Screenshot 2023-04-04 at 00 04 16

File: ofl/notosansmono/METADATA.pb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name: "Noto Sans Mono"
 designer: "Google"
 license: "OFL"
-category: "MONOSPACE"
+category: "SANS_SERIF"
```
Comment (m4rc1e) [inline ofl/notosansmono/METADATA.pb:4]: Is this category chance intentional?

**Comment by simoncozens** [comment]
> Seems to be caused by either the forward slash or fraction.

Nah, it's just Glyphs's wonderful habit of randomly moving components around when you open a file.

**Comment by simoncozens** [comment]
Updated

## Noto Sans Mono: Version 2.012 added

---
 8b0c380: [gftools-packager] Noto Sans Mono: Version 2.012 added

* Noto Sans Mono Version 2.012 taken from the upstream repo https://github.com/notofonts/latin-greek-cyrillic.git at commit https://github.com/notofonts/latin-greek-cyrillic/commit/.

**Review by m4rc1e — COMMENTED** [review]
Issue still present.

Screenshot 2023-04-04 at 17 06 48

**Comment by simoncozens** [comment]
Updated

## Noto Sans Mono: Version 2.013 added

---
 5d6219f: [gftools-packager] Noto Sans Mono: Version 2.013 added

* Noto Sans Mono Version 2.013 taken from the upstream repo https://github.com/notofonts/latin-greek-cyrillic.git at commit https://github.com/notofonts/latin-greek-cyrillic/commit/.

**Comment by moyogo** [comment]
@RosaWagner it's normal for this font. Many glyphs are double-width.

**Pull request merged**